### ISSUE TITLE
Application to home automation

### ISSUE THREAD

**Issue by robmarkcole** [wildtreetech/bayesian-optimisation#3]
Hi Tim
one challenge I have in home-automation is selecting the right priors for my [Bayesian sensors](https://www.home-assistant.io/components/binary_sensor.bayesian/), is that an application of interest? Background [here](https://github.com/robmarkcole/HASS-data-science/blob/master/Bayesian%20sensor%20tutorial.ipynb).
Cheers

**Comment by betatim** [comment]
Would tune `prob_given_true` and friends with this?

**Comment by robmarkcole** [comment]
Precisely. Some kind of GUI (Bokeh?) to explore and optimise the `prob_given_true/false` would be really interesting and useful
